### PR TITLE
update contract item for rendering messages

### LINKED ISSUE #33: Change the layout of the contract item component and add contract message sending feature
# Task Summary

Change the `ContractItem.vue` component layout so it is more intuitive for the user. Add intractability to the contract (sign & send contract message, fetch contract store value).

# Task List

- [x] remove the `Code bundle name` section since it is already rendered above the contract address. Make the copy icon size consistent https://github.com/PlasmNetwork/plasm-apps-vue/blob/45381bafb72eabba103bff0cae0b12a9dba092c2/src/components/dapps/ContractItem.vue#L87
remove-code-bundle-name

- [x] add a `export ABI` (or `download metadata`) button that lets the user download the `metadata.json` file from the contract item, instead of displaying the `json` string https://github.com/PlasmNetwork/plasm-apps-vue/blob/45381bafb72eabba103bff0cae0b12a9dba092c2/src/components/dapps/ContractItem.vue#L95
add-abi-download

- [x] display contract methods and stores in the contract item. We will need to fix the contract message component and abi hook to work on this https://github.com/PlasmNetwork/plasm-apps-vue/blob/45381bafb72eabba103bff0cae0b12a9dba092c2/src/components/dapps/ContractInfo.vue#L25
list-contract-methods

### PR DESCRIPTION
**Pull request by sirius651**
Fixed #33

### CODE REVIEW

**Comment by sirius651** [comment]
스크린샷 2021-06-30 오후 1 27 48

regarding `export ABI (or download metadata)`, I tried it, but it failure. In browser environment, there's a content size limitation on the default processor, so should consider using the server side rendering framework like nuxt or quasar. 
so I just changed the button to a `copy Abi` button to copy the abi

**Comment by sirius651** [comment]
regarding this `allow users to pass arguments for contract methods or fetch contract stores. We may want to create a contract modal that renders all interactable messages if we want this feature to be extendible. If this task takes too long, we'll separate this into its own issue and skip this one for the initial launch.`, it seems related with #32, @LRUki ongoing. so, I think it's rather to look at #32 together.  @hoonsubin

**Comment by hoonsubin** [comment]
> regarding this `allow users to pass arguments for contract methods or fetch contract stores. We may want to create a contract modal that renders all interactable messages if we want this feature to be extendible. If this task takes too long, we'll separate this into its own issue and skip this one for the initial launch.`, it seems related with #32, @LRUki ongoing. so, I think it's rather to look at #32 together. @hoonsubin

You're right. I'll move the subtask item in #33 regarding the contract message transaction to #32

File: src/components/dapps/ContractItem.vue [hunk 1]
```diff
@@ -79,25 +65,28 @@
...
-        </div>
-      </div>
-      <div>
-        <div class="text-xs text-gray-500 dark:text-darkGray-400">
-          Contract ABI
-        </div>
+        <div class="text-xs text-gray-500 dark:text-darkGray-400">Messages</div>
         <div class="text-xs text-blue-900 dark:text-darkGray-100">
-          {{ shortenAbi }}
+          <message
```
Comment (hoonsubin) [inline src/components/dapps/ContractItem.vue:70]: bandicam 2021-06-30 14-05-05-972
The contract message tooltip seems to cut off when it spans outside of the contract item component.
Related to this line https://github.com/PlasmNetwork/plasm-apps-vue/blob/45381bafb72eabba103bff0cae0b12a9dba092c2/src/components/dapps/Message.vue#L17

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/components/dapps/ContractItem.vue (hunk 1 above)
Comment (sirius651) [reply src/components/dapps/ContractItem.vue:70]: Resolved

**Review by hoonsubin — APPROVED** [review]
Thank you for the fix!
LGTM

**Pull request merged**